### DISCUSSION TITLE
USB Webcam not working

### DISCUSSION THREAD

**Discussion by DonChooluvit** [rudetrooper/Octoprint-Chituboard#34] [Q&A] [answered]
I have a Logitech C920 plugged into a USB hat on an rPi zero 2 w. The camera works in octoprint up to the point that the usb samba share is setup with chituboard. I tried all ports, and multiple cameras, no luck.

**Comment by rudetrooper** [comment, marked as answer]
It won't work because the Linux gadget subsystem that the USB share works on gets exclusive access to the USB OTG port. So the only option is to use a pi cam or arducam that makes use of the CSI port on the pi.

**Reply by DonChooluvit** [reply]
Copy that. Would it work with an rPi 4?

**Reply by rudetrooper** [reply]
Yeah, but the RPI4 would need to be powered via the GPIO.
